index.html

```diff
@@ -5186,7 +5182,7 @@ <h3>Content Integrity Protection</h3>
   "type": ["VerifiableCredential", "ExampleAlumniCredential"],
   "credentialSubject": {
     "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "image": <span class="highlight">"ipfs:/ipfs/QmXfrS3pHerg44zzK6QKQj6JDk8H6cMtQS7pdXbohwNQfK/image"</span>,
+    "image": <span class="highlight">"https://example.com/image"</span>,
```

Comment:
```suggestion
    "image": <span class="highlight">"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+M9QDwADhgGAWjR9awAAAABJRU5ErkJggg=="</span>,
```

Reply:
^ in case folks prefer to not encourage tracking via HTTPS urls, I provide the following small, but valid data uri.